### ISSUE TITLE
Adding BaltimoreCyberTrustRoot CA for Azure Database

### ISSUE THREAD

**Issue by shibayan** [microsoft/Oryx#853]
### System Package Request

If the Root CA, which is required to connect to the Azure Database, is installed on runtime image, it makes it easier to use and helps to build secure applications.

- https://docs.microsoft.com/en-us/azure/mysql/howto-configure-ssl#step-1-obtain-ssl-certificate
- https://docs.microsoft.com/en-us/azure/postgresql/concepts-ssl-connection-security#applications-that-require-certificate-verification-for-tls-connectivity

**Comment by kichalla** [comment]
Thanks for the details @shibayan. We will look into it.

**Comment by kichalla** [comment]
@shibayan Which runtime image are you referring to? I just checked few versions of runtime images for each platform (ex: Node, Python etc.) and I see that the `BaltimoreCyberTrustRoot CA` is already part of the images. 
FYI...

```
C:\> docker run -it oryx/python:3.7                                                                                                                                                           root@8271b82ef781:/# find / -iname *baltimore*.pem
/etc/ssl/certs/Baltimore_CyberTrust_Root.pem
root@8271b82ef781:/# exit
exit
C:\> docker run -it oryx/python:3.9                                                                                                                                                           root@5e4f684ff9c1:/# find / -iname *baltimore*.pem
/etc/ssl/certs/Baltimore_CyberTrust_Root.pem
root@5e4f684ff9c1:/# exit
exit
C:\> docker run --entrypoint bash -it oryx/node:14                                                                                                                                            root@c39f269ceed5:/# find / -iname *baltimore*.pem
/etc/ssl/certs/Baltimore_CyberTrust_Root.pem
root@c39f269ceed5:/# exit
exit
C:\> docker run --entrypoint bash -it oryx/node:12                                                                                                                                            root@caedd45e5c64:/# find / -iname *baltimore*.pem
/etc/ssl/certs/Baltimore_CyberTrust_Root.pem
root@caedd45e5c64:/# exit
exit
C:\> docker run --entrypoint bash -it oryx/dotnetcore:5.0                                                                                                                                     root@7053bd6efc11:/# find / -iname *baltimore*.pem
/etc/ssl/certs/Baltimore_CyberTrust_Root.pem
root@7053bd6efc11:/# exit
exit
C:\> docker run --entrypoint bash -it oryx/dotnetcore:3.1                                                                                                                                     root@981b0b82ceeb:/# find / -iname *baltimore*.pem
/etc/ssl/certs/Baltimore_CyberTrust_Root.pem
root@981b0b82ceeb:/# exit
exit
C:\> docker run --entrypoint bash -it oryx/php:7.3                                                                                                                                            root@adc220fd6077:/var/www/html# find / -iname *baltimore*.pem
/etc/ssl/certs/Baltimore_CyberTrust_Root.pem
root@adc220fd6077:/var/www/html# exit
exit
C:\> docker run --entrypoint bash -it oryx/php:7.4                                                                                                                                            root@d9f0b72a029c:/var/www/html# find / -iname *baltimore*.pem
/etc/ssl/certs/Baltimore_CyberTrust_Root.pem
root@d9f0b72a029c:/var/www/html# exit
exit
C:\>
```

**Comment by shibayan** [comment]
@kichalla Thank you for sharing the information. Reading this document, I thought I needed to add a Root CA, but now it seems to work without adding a CA.
https://docs.microsoft.com/en-us/azure/app-service/tutorial-php-mysql-app?pivots=platform-linux#configure-tlsssl-certificate

I thought the installed `BaltimoreCyberTrustRoot CA` was an old version, but I checked the file path certificate you gave me, and it's the latest version.
